update_uri.py

```
@@ -0,0 +1,40 @@
+import re
+
+with open("crates/perl-module-resolution-uri/src/lib.rs", "r") as f:
```

Ruff (0.15.9)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)